src/TheBuild/Acquia/GetLatestBackupTask.php

```diff
@@ -361,10 +196,15 @@ protected function validate() {
     if (empty($this->database)) {
       $this->database = $this->site;
     }
+    // Check .env file exists.
+    if (!file_exists(".env")) {
+      throw new \BuildException(".env file is needed in the root of project with credentials see .env.example.");
```

Comment:
This is checking for the `.env` file in the root, but it actually needs to exist in the `.ddev` directory for the environment variables to be created. I'm going to remove this, because the API already checks for the presence of these variables and fails accordingly:
```
example > acquia-get-backup:

[getacquiabackup] Couldn't find ACQUIA_CLOUD_API_KEY env variable.
[getacquiabackup] Couldn't find ACQUIA_CLOUD_API_SECRET env variable.

BUILD FAILED
```